packages/oas-to-har/__tests__/lib/style-formatting/index.test.js

```diff
@@ -0,0 +1,948 @@
+      ],
+      [
+        'should support matrix path styles styles for exploded array input',
+        paramExplode,
+        { path: { color: arrayInput } },
+        `https://example.com/style-path/${semicolon}color${equals}blue${semicolon}color${equals}black${semicolon}color${equals}brown`,
+      ],
+      [
+        'should support matrix path styles styles for non exploded object input',
+        paramNoExplode,
+        { path: { color: objectInput } },
+        `https://example.com/style-path/${semicolon}color${equals}R${comma}100${comma}G${comma}200${comma}B${comma}150`,
+      ],
+      [
+        'should support matrix path styles styles for exploded object input',
+        paramExplode,
+        { path: { color: objectInput } },
+        `https://example.com/style-path/${semicolon}R${equals}100${semicolon}G${equals}200${semicolon}B${equals}150`,
+      ],
+    ])('%s', async (testCase, operation = {}, values = {}, expectedUrl) => {
+      const har = oasToHar(
+        oas,
+        {
+          path: '/style-path/{color}',
+          method: 'get',
+          ...operation,
+        },
+        values
+      );
+
+      await expect(har).toBeAValidHAR();
+
+      expect(har.log.entries[0].request.url).toStrictEqual(expectedUrl);
+    });
+  });
+
+  describe('label path', () => {
+    const paramNoExplode = {
+      parameters: [
+        {
+          name: 'color',
+          in: 'path',
+          style: 'label',
+          explode: false,
+        },
+      ],
+    };
+
+    const paramExplode = {
+      parameters: [
+        {
+          name: 'color',
+          in: 'path',
+          style: 'label',
+          explode: true,
+        },
+      ],
+    };
+
+    it.each([
+      [
+        'should support label path styles non exploded empty input',
+        paramNoExplode,
+        { path: { color: emptyInput } },
+        'https://example.com/style-path/.',
+      ],
+      [
+        'should support label path styles styles for exploded empty input',
+        paramExplode,
+        { path: { color: emptyInput } },
+        'https://example.com/style-path/.',
+      ],
+      [
+        'should support label path styles styles for non exploded string input',
+        paramNoExplode,
+        { path: { color: stringInput } },
+        'https://example.com/style-path/.blue',
+      ],
+      [
+        'should support label path styles styles for exploded string input',
+        paramExplode,
+        { path: { color: stringInput } },
+        'https://example.com/style-path/.blue',
+      ],
+      [
+        'should support label path styles styles for non exploded array input',
+        paramNoExplode,
+        { path: { color: arrayInput } },
+        'https://example.com/style-path/.blue.black.brown',
+      ],
+      [
+        'should support label path styles styles for exploded array input',
+        paramExplode,
+        { path: { color: arrayInput } },
+        'https://example.com/style-path/.blue.black.brown',
```

Comment:
I have absolutely no idea why anyone would ever want to do this 🤷‍♂️

Reply:
I would hope anyone doing anything like this uses query params instead of the path. It's definitely bizarre.